django/db/models/functions.py

```diff
@@ -47,7 +47,8 @@ def as_sqlite(self, compiler, connection):
         return super(ConcatPair, self).as_sql(compiler, connection)
 
     def as_mysql(self, compiler, connection):
-        self.coalesce()
+        self.function = 'CONCAT_WS'
+        self.template = '%(function)s("", %(expressions)s)'
```

Comment:
Does the use of double quotes for the separator argument matter here? I'm used to most string values having to use single quotes, but I'm not a mysql user so I'm not entirely sure it matters. I did a quick search and it appears that mysql supports double or single quotes for string literals.